### DISCUSSION TITLE
Compatibility with S3 driver

### DISCUSSION THREAD

**Discussion by grpaiva** [konnco/filament-import#85] [Q&A]
Thanks for the package, great work!

I'm using Laravel Vapor so saving temporary files in Lambda's execution environment is not a good idea.

I've noticed there's a `temporary_files` config, tried to switch to `s3`:

```{php}
return [
    ...
    'temporary_files' => [
        'disk' => 's3',
        'directory' => 'tmp',
    ],
];
```

It kinda worked, file is being uploaded to my S3 bucket, but when trying to save I think it's still looking for it in my local disk, so I get this error:

```
Could not find zip member zip:///var/www/html/storage/framework/cache/laravel-excel/laravel-excel-bfABVWfqoipFNyzb4oSuKgKo9RLTZZxM.xlsx#_rels/.rels
```

Any ideas?

Cheers

**Comment by elmudometal** [comment]
I have the same problem
